### DISCUSSION TITLE
Welcome to lettura Discussions!

### DISCUSSION THREAD

**Discussion by zhanglun** [zhanglun/lettura#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

👋 欢迎！
  我们将讨论区作为与社区其他成员联系的场所。我们希望您
  * 提出您想知道的问题。
  * 分享想法。
  * 与其他社区成员互动。
  * 欢迎他人并持开放态度。请记住，这是一个我们
  共同建立的社区。

  要想开始，请在下面评论中介绍自己，并告诉我们您在这个社区做什么。

👋 ようこそ！
  ディスカッションは、私たちのコミュニティの他のメンバーとつながるための場所として使用しています。ぜひご利用ください：
  * 疑問に思っていることを質問してください。
  * アイデアを共有する。
  * 他のコミュニティメンバーと交流する。
  * 他の人を歓迎し、オープンマインドであること。このコミュニティは私たちが
  一緒に作り上げるコミュニティであることを忘れないでください。

  まずは、自己紹介とこのコミュニティで何をするのかについて教えてください。

**Comment by openersoft** [comment]
Awesome project! Any plans for mobile version?

**Comment by YugandharrPatil** [comment]
how do you find subscriptions? the search funtionality doesn't work for me (i'm using the aur package on arch: https://aur.archlinux.org/packages/lettura-bin

is this something you officially support?
